config/examples/Elegoo/Neptune 2/Configuration.h

```diff
@@ -69,9 +69,60 @@
 // @section info
 
 // Author info of this build printed to the host during boot and M115
-#define STRING_CONFIG_H_AUTHOR "(EvilGremlin)" // Who made the changes.
+#define STRING_CONFIG_H_AUTHOR "(just-trey, mlee12382, and EvilGremlin, Neptune 2 - Marlin 2 Color UI)" // Who made the changes.
```

Comment:
A string this long may be problematic. Watch carefully for crashes and buffer overflows.